19 add css

bricknj opened the pull request:
Created a css file to clear all default browser styling and then add in all custom styling.

nknaian commented on musicrecs/static/style.css:
```diff
@@ -0,0 +1,197 @@
+/* PAGE round/listen_phase.html */
+
+/* ERROR - This is not currently functioning, but it seemed to
+have the same issue in the html code */
```
What wasn't working before?

bricknj replied:
That section of code was flagged as an error for me before I touched it. For scaling the album art. I think the current issue in css is because I'm using {{ }} within another set of { }. I think that is probably just a matter of finding the appropriate css symbol to use.

nknaian replied:
when you say flagged do mean a warning by vscode? I don't see any problems visually when I tested this out

nknaian commented on musicrecs/static/style.css:
```diff
@@ -0,0 +1,197 @@
+/* Broad Styles */
+h2 {
+    margin-bottom: 20px;
+}
```
Maybe we need to add more style to other heading levels. I was looking at the submit heading (which is level h4) and it looks a bit squished

![image](https://user-images.githubusercontent.com/13397301/108632533-6f0ecc80-743d-11eb-9bfe-67c94ab7a0a2.png)

nknaian commented on musicrecs/static/style.css:
```diff
@@ -0,0 +1,197 @@
+    font-style: italic;
+}
+
+/* ID Styles (organized by page) */
```
nice organization 👍

nknaian commented on musicrecs/static/style.css:
```diff
@@ -0,0 +1,197 @@
+v1.6.1
+Last Updated: 2010-09-17
+Author: Richard Clark - http://richclarkdesign.com 
+Twitter: @rich_clark
```
Do we have to credit rich clark here? Haha

bricknj replied:
I'm just trying to stay in good standing with the Open Source Administration.

nknaian replied:
fair enough

nknaian commented on musicrecs/templates/index.html:
```diff
@@ -2,39 +2,41 @@
-        <p style="margin-left: 20px;"><i>Revealed Phase</i>: It is revealed which participants recommended what.</p>
+        <p class="p_index_phase"><i>Submit Phase</i>: Participants submit spotify music links of their recommendations.</p>
+        <p class="p_index_phase"><i>Listen Phase</i>: Participants listen to the music and consider who may have submitted what.</p>
+        <p class="p_index_phase"><i>Revealed Phase</i>: It is revealed which participants recommended what.</p>
```
The text on this page looks too cramped to me now. Maybe your styling is revealing that having separate paragraphs for each line is not a good idea haha.

nknaian replied:
![image](https://user-images.githubusercontent.com/13397301/108632818-daa56980-743e-11eb-9b9a-b701db2dfbc9.png)

nknaian replied:
![image](https://user-images.githubusercontent.com/13397301/108633087-5b189a00-7440-11eb-80ab-a284ba39ad56.png)

nknaian replied:
But now that I'm looking at how it compares to the old one, that old one is too spaced out. I think we need a goldilocks spacing

bricknj replied:
Yeah, this is a good point to note. I think it will be a matter of playing around with the spacing. But I see no reason that can't all be formatted into one p for all three of those.

nknaian reviewed (changes requested):
Looking good, this makes each html page a lot simpler. Organizing the style sheet into things that are general and things that are specific to each page is super helpful.

There are a few things that are looking squished/cramped though that I'd like you to figure out a solution to.
And what's wrong with the album art stuff? Did something I wrote styled into html not actually work and I never noticed?

bricknj commented:
I added styling for the master updates, cleaned up the phase paragraph spacing, added spacing defaults for the headers we use, and removed the reset stylesheet as it was removing Bootstrap styling.

nknaian reviewed (changes requested):
Just get rid of the 'todo.txt' and then I'll give you the go-ahead to merge! Looks good to me, keeping the bootstrap settings seemed to resolve lots of my concerns

nknaian reviewed (approved):
Good to go, squash and merge it!
